FAQ.md

```diff
@@ -15,4 +15,8 @@ Such as:
 * I'm getting an annoying SSL warning message
 * "No operations allowed after connection closed" error
 
-Please see [here](https://github.com/lucko/LuckPerms/wiki/Storage-system-errors).
\ No newline at end of file
+Please see [here](https://github.com/lucko/LuckPerms/wiki/Storage-system-errors).
+
+### I use SpongeForge and when I install LuckPerms do plugins no longer work
+This is an intentional behaviour of SpongeForge. As soon as a permission plugin, in this case LuckPerms, is installed will it disable the OP system.  
```

Comment:
Shouldn't it be "is installed it will" here?

Reply:
Not quite sure